config/sync/core.entity_form_display.paragraph.paragraph_access_permission.default.yml

```diff
@@ -0,0 +1,36 @@
+uuid: 7dc47fad-a108-441b-846d-0e324df3db8f
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.paragraph.paragraph_access_permission.field_acc_perm_accesses
```

Comment:
Sugiro tirar o acc e deixar permission, palavra toda.

Lembra que para o graphql é importante seguir os nomes da entidade para funcionar como plug and play?

Reply:
OK!